rca/project_styleguide/templates/patterns/organisms/programme_tabs/experience.html

```diff
@@ -0,0 +1,79 @@
+                        </div>
+                    </div>
+                {% endif %}
+                
+                {% if page.experience_cta_link %}
+                    <div class="overview__item">
+                        <div class="overview__item-action">
+                            {% for block in page.experience_cta_link %}
+                                {% include_block block %}
+                            {% endfor %}
+                        </div>
+                    </div>
+                {% endif %}
+            </div>
+        {% endif %}
+
+        {% if page.experience_content %}
+            {% for block in page.experience_content %}
+                {% if block.block_type == "social_embeds" or block.block_type == "story" %}
+                    <div class="section__header grid">
+                        <h3 class="section__heading section__heading--secondary heading heading--three">{{ block.value.title }}</h3>
+                    </div>
+                    <div class="section__row grid">
+                        <div class="results layout__start-one layout__span-two layout__@large-start-two layout__@large-span-three">
+                            {% if block.block_type == "social_embeds" %}
+                                {% for individual_embed in block.value.embeds %}
+                                    <div class="results__item layout__span-two layout__@medium-span-one social-embeds__item">
+                                        <div class="embed-container social-embeds__container">
+                                            {{ individual_embed.embed.html|safe }}
+                                            <div class="social-embeds__caption">
+                                                <h4 class="heading heading--five social-embeds__heading">
+                                                    <a href="{{ individual_embed.embed.url }}" class="link link--tertiary">{{ individual_embed.caption }}</a>
+                                                </h4>
+                                                <p class="social-embeds__source">
+                                                    From {{ individual_embed.embed_source }}
+                                                </p>
+                                            </div>
+                                        </div>
+                                    </div>
+                                {% endfor %}
+                            {% elif block.block_type == "story" %}
+                                {% for story in block.value.stories %}
+                                    <div class="results__item layout__span-two layout__@medium-span-one story-sections__item">
+                                        {% image story.image fill-400x300 as story_image %}
+                                        <img src="{{ story_image.url }}" alt="{{ story.value.title }}" class="story-sections__image">
```

Comment:
It's worth using the template `patterns/atoms/image/image--lazyload.html` for this to make loading the page a bit more efficient. There's a lot of samples that use it e.g. `{% include "patterns/atoms/image/image--lazyload.html" with image_small=image_small width=829 height=585 image_large=image_large classList='slide__image' %}`.